package.json

```diff
@@ -95,14 +105,17 @@
     "presets": [
       "@babel/preset-env",
       "@babel/preset-react"
+    ],
+    "plugins": [
+      "@babel/plugin-transform-runtime"
     ]
   },
   "prettier": {
     "trailingComma": "es5",
```

Comment:
I understood that this is now the default and can be removed.

Reply:
Yes, I'll remove this.